todo_app/integration_test.py

```diff
@@ -26,27 +27,58 @@ def client():
 @patch('requests.get')
 def test_index_page(mock_get_requests, client):
 # Replace call to requests.get(url) with our own function
-    mock_get_requests.side_effect = mock_get_lists
-    test_board_id = '6052828032dafa5707bf5reg'
-    url = f'https://api.trello.com/1/boards/{test_board_id}/lists'
-    response = client.get(url)
-    
+     mock_get_requests.side_effect = mock_get_lists
+     test_board_id = '6052828032dafa5707bf5reg'
```

Comment:
We never use this board id so we can delete it here

Reply:
Updated,thank you